packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -107,6 +113,33 @@ export const indexPage =
         );
       }
 
+      if (serverlessFuncs.length) {
+        const systemEventFuncs: Record<string, PluginSource> = {};
+        const apiFuncs: Record<string, PluginSource> = {};
```

Comment:
Nit: Can we call this `httpFuncs`? Product is moving away from the "Api" verbiage in favor of "Http".